packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -17,34 +17,90 @@
 /**
  * External dependencies
  */
+import cloneDeep from 'clone-deep';
 import { useEffect, useRef } from '@web-stories-wp/react';
 
 /**
  * Internal dependencies
  */
 import { useHistory } from '../../history';
+import deleteNestedKeys from '../utils/deleteNestedKeys';
+
+// Changes to these properties of elements do not create a new history entry.
+const ELEMENT_PROPS_TO_IGNORE = [
+  'resource.baseColor',
+  'resource.isMuted',
+  'resource.poster',
+  'resource.posterId',
```

Comment:
Hm, I'm not sure what would be the best way to test this change 🤔 

Since there are still the transient props then history is still being mutated in a lot of cases.

E.g. on `main`, when I drag a video from my computer directly onto the Page, it takes 7 clicks of Undo to get back to the state without video. On this PR, however, it takes "only" 4 clicks -- there are still the transient props changes being tracked there.

Not sure what would be a good way to QA this though. Probably a combination of #9785 and this PR would be easier to test.

Thoughts here? cc @spacedmonkey @swissspidy

Reply:
We can certainly wait until #9785 lands, it's fairly close.

After that it should be easier to test.